README.md

```diff
@@ -1338,6 +1338,7 @@ An adapter can implement any or all of the following methods:
 1. diff: Returns the changeset in the desired format given the changeset in the original format
 2. load_changeset: Returns the changeset for a given version object
 3. where_object_changes: Returns the records resulting from the given hash of attributes.
+4. where_object_changes_from: Returns the records resulting from the given hash of attributes where the attributes changed *from* the provided value(s).
```

Comment:
I pushed a commit adding this information. @jaredbeck, do you think there's more we need to do to ensure custom adapters will work? Is there a list of known custom adapters (beyond the one mentioned in the README) that we know of?

Should we drop issues in those repos, alerting them to the additional surface area to the API being added here?